docs/source/contributing.rst

```diff
@@ -37,8 +37,8 @@ To install ``soundata`` for development purposes:
     .. code-block:: console
 
         pip install .
-        pip install .[tests]
-        pip install .[docs]
+        pip install ."[tests]"
```

Comment:
Cool :) But the rest of the changes will be already applied in #114 right? Or vice versa.